app/lib/tcomb-form/factories/LocalImageFactory.js

```diff
@@ -0,0 +1,33 @@
+import t from 'tcomb-form-native';
+import ImagePicker from 'react-native-image-picker';
+
+import localImageTemplate from '../templates/LocalImageTemplate';
+
+const Component = t.form.Component;
+
+export default class LocalImageFactory extends Component {
```

Comment:
It is specific for library,  has concrete purpose there - factory. Also I think it could possibly be extracted from the project
So I decided to extract it somewhere